src/components/PostList/PostList.jsx

```diff
@@ -1 +1,10 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo/PostInfo';
```

Comment:
You have an index.js file in each folder, so you can just write `'../PostInfo'` (same for the others)